keyboards/kumo/keymaps/LEdiodes/keymap.c

```diff
@@ -0,0 +1,212 @@
+// Normal + arrow layout for MiniVan Kumo
+// Created by LEdiodes on 5-2-19
+
+#include "kumo.h"
+#include "action_layer.h"
```

Comment:
`#include QMK_KEYBOARD_H` includes this file automatically. This line should be deleted.

```suggestion
```